src/index.html

```diff
@@ -1,19 +1,315 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width,
+  <meta name="viewport" content="width=device-width,
           user-scalable=no,
           initial-scale=1.0,
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
   <title>Document</title>
   <link rel="stylesheet" href="./style.css">
 </head>
+
 <body>
 
-<script type="text/javascript" src="./main.js"></script>
+  <header class="header">
+    <a href="#" class="logo-link"><img src="./images/logo.png" class="navbar-brand" alt=""></a>
+    <nav class="navbar">
+      <ul class="nav-links">
+        <li class="nav-li">
+          <a href="#" class="is-active nav-link">apple</a>
+        </li>
+        <li class="nav-li">
+          <a href="#" class="nav-link">samsung</a>
+        </li>
+        <li class="nav-li">
+          <a href="#" class="nav-link">смартфоны и телефоны</a>
+        </li>
+        <li class="nav-li">
+          <a href="#" data-qa="nav-hover" class="nav-link is-hovered">ноутбуки и компьютеры</a>
+        </li>
+        <li class="nav-li">
+          <a href="#" class="nav-link">гаджеты</a>
+        </li>
+        <li class="nav-li">
+          <a href="#" class="nav-link">планшеты</a>
+        </li>
+        <li class="nav-li">
+          <a href="#" class="nav-link">фото</a>
+        </li>
+        <li class="nav-li">
+          <a href="#" class="nav-link">видео</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+  <main class="catalog">
+    <div class="list-of-items">
+
+      <div class="product-item">
+
+        <div class="imac-card" data-qa="card">
+          <a href="#">
+            <img class="imac-img" src="./images/imac.jpeg" alt="Моноблок APPLE A1419 iMac 27"
```

Comment:
`imac-img` => `product-image`
```suggestion
            <img
              class="product-image"
              src="./images/imac.jpeg"
              alt="Моноблок APPLE A1419 iMac 27"
              title="Моноблок APPLE A1419 iMac 27"
            />
```